compositions/upbound-aws-provider/aurora/aurora.yaml

```diff
@@ -0,0 +1,564 @@
+          toFieldPath: spec.forProvider.idleClientTimeout
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.proxyConfig.proxyRoleArn
+          toFieldPath: spec.forProvider.roleArn
+        - fromFieldPath: spec.networkConfig.subnetIds
+          toFieldPath: spec.forProvider.vpcSubnetIds
+        - type: FromCompositeFieldPath
+          fromFieldPath: status.clusterMasterSecretArn
+          toFieldPath: spec.forProvider.auth[0].secretArn
+        - type: FromCompositeFieldPath
+          fromFieldPath: status.securityGroupId
+          toFieldPath: spec.forProvider.vpcSecurityGroupIds[0]  
+        - type: ToCompositeFieldPath
+          fromFieldPath: status.atProvider.arn
+          toFieldPath: status.proxyArn
+        - type: ToCompositeFieldPath
+          fromFieldPath: status.atProvider.endpoint
+          toFieldPath: status.proxyEndpoint
+    - name: aurora-cluster-proxy-default-tg
+      base:
+        apiVersion: rds.aws.upbound.io/v1beta1
+        kind: ProxyDefaultTargetGroup
+        metadata:
+          name: aurora-proxy-default-tg
+        spec:
+          forProvider:
+            connectionPoolConfig:
+              - connectionBorrowTimeout: 10
+                maxConnectionsPercent: 50
+                maxIdleConnectionsPercent: 50
+            dbProxyNameSelector:
+              matchControllerRef: true
+      patches:
+        - type: PatchSet
+          patchSetName: common-fields
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.proxyConfig.connectionBorrowTimeout
+          toFieldPath: spec.forProvider.connectionPoolConfig[0].connectionBorrowTimeout
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.proxyConfig.maxConnectionsPercent
+          toFieldPath: spec.forProvider.connectionPoolConfig[0].maxConnectionsPercent  
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.proxyConfig.maxIdleConnectionsPercent
+          toFieldPath: spec.forProvider.connectionPoolConfig[0].maxIdleConnectionsPercent
+        - type: ToCompositeFieldPath
+          fromFieldPath: status.atProvider.name
+          toFieldPath: status.defaultProxyTgName 
+    - name: aurora-cluster-proxy-target  
+      base:
+        apiVersion: rds.aws.upbound.io/v1beta1
+        kind: ProxyTarget
+        metadata:
+          name: aurora-proxy-target
+        spec:
+          forProvider:
+            dbClusterIdentifier: proxy-db-sg
+            dbProxyNameSelector:
+              matchControllerRef: true
+            targetGroupName: default   
+      patches:
+        - type: PatchSet
+          patchSetName: common-fields
+        - type: FromCompositeFieldPath
+          fromFieldPath: status.defaultProxyTgName
+          toFieldPath: spec.forProvider.targetGroupName
+        - type: FromCompositeFieldPath
+          fromFieldPath: status.clusterIdentifier
+          toFieldPath: spec.forProvider.dbClusterIdentifier
```

Comment:
You mentioned the secrets for connection should go to secrets manager but I don't see them in the composition.

Reply:
What I was mentioning is that the db user and password is managed by secret manager. We are setting a parameter for the cluster  "manageMasterUserPassword" as  true which creates a secret manager and send the credential to that secret manager.